yarn-project/end-to-end/src/e2e_rollup_native_asset_contract.test.ts

```diff
@@ -0,0 +1,218 @@
+import { AztecNode, getConfigEnvVars } from '@aztec/aztec-node';
+import { AztecAddress, AztecRPCServer, Contract, ContractDeployer, EthAddress, TxStatus } from '@aztec/aztec.js';
+import { RollupNativeAssetContractAbi } from '@aztec/noir-contracts/examples';
+
+import { HDAccount, mnemonicToAccount } from 'viem/accounts';
+import { createAztecRpcServer } from './create_aztec_rpc_client.js';
+import { deployL1Contract, deployL1Contracts } from './deploy_l1_contracts.js';
+import { createDebugLogger } from '@aztec/foundation/log';
+import { Fr, Point } from '@aztec/foundation/fields';
+import { toBigIntBE, toBufferBE } from '@aztec/foundation/bigint-buffer';
+import { fr } from '@aztec/circuits.js/factories';
+import { sha256 } from '@aztec/foundation/crypto';
+import { OutboxAbi, RollupNativeAssetAbi, RollupNativeAssetBytecode } from '@aztec/l1-artifacts';
+import {
+  GetContractReturnType,
+  PublicClient,
+  HttpTransport,
+  Chain,
+  getContract,
+  createPublicClient,
+  http,
+  getAddress,
+  createWalletClient,
+  Address,
+} from 'viem';
+import { foundry } from 'viem/chains';
+
+const MNEMONIC = 'test test test test test test test test test test test junk';
```

Comment:
to avoid a merge conflict with Sean's work, you can create a `fixture.ts` and import MNEMONIC from there! [Ref](https://github.com/AztecProtocol/aztec-packages/pull/578/files#diff-f7a480d6e733bbe902e8b8dec95d1ef7e47759b1d934636a392d67adeb0635abR1)

Reply:
its all good im ok with refactoring it when i merge master